src/webui/features/ReactionDetails.tsx

```diff
@@ -0,0 +1,80 @@
+
+import { useCallback } from 'react';
+
+import type { AggregatedData as PostView } from '^/domain/post/aggregate/types';
+import type { AggregatedData as ReactionView } from '^/domain/reaction/aggregate/types';
+
+import { ConfirmationPanel, LoadingContainer, PostDetailsPanel, ReactionLargePanel, SingleReactionRow } from '../components';
+import { useAppContext } from '../contexts';
+import { Column, Ruler } from '../designsystem';
+import useEstablishRelation from './hooks/useEstablishRelation';
```

Comment:
```suggestion
import { Column, Ruler } from '../designsystem';

import useEstablishRelation from './hooks/useEstablishRelation';
```

Missing enter between the external and local imports.